guides/release/upgrading/current-edition/index.md

```diff
@@ -111,12 +110,13 @@ A fully-Octane app has the following configuration in `config/optional-features.
 ```json
 {
   "application-template-wrapper": false,
+  "default-async-observers": true,
```

Comment:
This seems a little dubious ("fully-Octane" 🤔) but so does the jquery thing below, so seems fine!